schema/appdal/trigger.schema.xml

```diff
@@ -99,19 +99,52 @@
   <attribute name="trigger_types" type="u32" is-multi-value="yes" init-value="3,7" is-not-null="yes"/>
   <attribute name="trigger_intervals" type="u32" is-multi-value="yes" init-value="62500000,62500000" is-not-null="yes"/>
   <attribute name="clock_frequency_hz" type="u32" init-value="62500000"/>
-  <attribute name="timestamp_method" type="enum" range="kTimeSync,kSystemClock" init-value="kTimeSync" is-not-null="yes"/>
  </class>
 
- <class name="ModuleLevelTrigger">
+ <class name="DataSubscriber" is-abstract="no">
   <superclass name="DaqModule"/>
+  <relationship name="configuration" class-type="DataReaderConf" low-cc="one" high-cc="one" is-composite="no" is-exclusive="no" is-dependent="no"/>
+ </class>
+ 
+ <class name="HSI2TCTranslatorConf">
+  <superclass name="DataReaderConf"/>
+  <relationship name="signals" class-type="HSISignalWindow" low-cc="one" high-cc="many" is-composite="no" is-exclusive="no" is-dependent="no"/>
+ </class>
+
+  <class name="HSISignalWindow">
+  <attribute name="signal_type" type="u32" init-value="0" is-not-null="yes"/>
+  <attribute name="time_before" type="u32" init-value="10000" is-not-null="yes"/>
+  <attribute name="time_after" type="u32" init-value="20000" is-not-null="yes"/>
+ </class>
+
+ <!--
```

Comment:
The genconfig code generation still tries to build this commented out class and fails. Maybe we need to update `genconfig` but the schema editors will remove the comment if anybody edits the file anyway so it's best not to rely on comments.

Reply:
Actually, I just checked, it's not genconfig itself that ignores the comment delimiters, it's the `daq_oks_codegen` function in daq-cmake getting the list of classes through a grep.